portaljs-pages/page-admin-case-visualization/src/index.html

```diff
@@ -0,0 +1,27 @@
+<!DOCTYPE html>
+<html xmlns="http://www.w3.org/1999/html">
+<head>
+    <script>
+        function getUrlParameter(name) {
+            name = name.replace(/[\[]/, '\\[').replace(/[\]]/, '\\]');
+            var regex = new RegExp('[\\?&]' + name + '=([^&#]*)');
+            var results = regex.exec(location.search);
+            return results === null ? '' : decodeURIComponent(results[1].replace(/\+/g, ' '));
+        };
+
+        function getPortalUrl() {
+            var locationHref = window.location.href;
+            var indexOfPortal = locationHref.indexOf('/portal/');
+            if (indexOfPortal >= 0) {
+                return locationHref.substring(0, indexOfPortal);
+            }
+        };
+
+        let processId = getUrlParameter("id");
```

Comment:
maybe we should change the name of this parameter

Reply:
Do you mean id or processId?

Reply:
If id, then we would also need to change the page.